src/constant_time.rs

```diff
@@ -40,6 +40,7 @@ mod tests {
     use crate::{bssl, error};
 
     #[test]
+    #[cfg(not(target_arch = "s390x"))] // Triggers a qemu bug before 8.0.3
```

Comment:
OK, this is not what I was expecting. I thought we'd have an `#ifdef` around a line or two within `constant_time_test.cc`.

Maybe instead we should change ci.yml so that we have something like:
```
          - target: s390x-unknown-linux-gnu
            host_os: ubuntu-22.04
            # XXX: `constant_time_test` fails when run under QEMU 8.0.2 and earlier
            # (https://link that you shared with me in your earlier comment).
            cargo_options:  -- --skip constant_time_test
```

This way people would not ever run into the issue on bare metal testing and I could also play with it locally. WDYT?

Reply:
I see, this is indeed nicer.  I've changed the PR accordingly.